src/electron/platform/android/setup/steps/prompt-configuring-port-forwarding-failed.ts

```diff
@@ -3,13 +3,11 @@
 
 import { AndroidSetupStepConfig } from 'electron/platform/android/setup/android-setup-steps-configs';
 
-export const promptConfiguringPortForwardingFailed: AndroidSetupStepConfig = deps => {
+export const promptConfiguringPortForwardingFailed: AndroidSetupStepConfig = stepTransition => {
     return {
         actions: {
-            cancel: () => {
-                deps.stepTransition('prompt-choose-device');
-            },
-            next: () => deps.stepTransition('configuring-port-forwarding'),
+            cancel: () => stepTransition('prompt-choose-device'),
```

Comment:
You've added cancel and next, but left the previous versions (using deps) on lines 10 & 12. Seems odd...